Tests/xcprojTests/XCSchemeSpec.swift

```diff
@@ -60,12 +60,21 @@ final class XCSchemeIntegrationSpec: XCTestCase {
         XCTAssertEqual(scheme.testAction?.macroExpansion.blueprintName, "iOS")
         XCTAssertEqual(scheme.testAction?.macroExpansion.referencedContainer, "container:Project.xcodeproj")
         XCTAssertEqual(scheme.testAction?.macroExpansion.referencedContainer, "container:Project.xcodeproj")
+        XCTAssertNotNil(scheme.testAction?.commandlineArguments)
+        if let testCLIArgs = scheme.testAction?.commandlineArguments {
```

Comment:
@rahul-malik I think we can simplify this by adding a custom assertion:

```swift
let commandLineArguments = try XCTAssertNotNilAndUnwrap(scheme.testAction?.commandlineArguments)
XCTAssertTrue(commandLineArguments.count > 0)
XCTAssertEqual(commandLineArguments[0].name, "MyTestArgument")
XCTAssertTrue(commandLineArguments[0].enabled)
```
By doing that we avoid the testing in our tests. What do you think?

Reply:
Same comment applies to the other tests in this file.